### DISCUSSION TITLE
How can I get only the latest relation created?

### DISCUSSION THREAD

**Discussion by ranst91** [prisma/prisma#8226] [Q&A]
Considering the following code:
```ts
prisma.user.update({
      where: { id: userId },
      data: {
        posts: {
          create: [post],
        },
      },
      include: {
        posts: {
          ... this is where the magic should happen
        }
      }
    });
```
Once the post is created, I want to include only this post in the returned result. Instead of getting all the other users posts as well. So just the one I just created.
How can this be done?

PS - the post has nothing unique other than the ID that will be assigned to it (which I don't know yet ofc). So I can't think of anything which is not the id to put under `posts: { where: ... }`

Looking at the docs I came up with nothing ofc :(

**Comment by ryands17** [comment]
@ranst91 👋 

As this returns all the relations, you would need to query explicitly for the latest one as follows:

```ts
prisma.user.update({
      where: { id: userId },
      data: {
        posts: {
          create: [post],
        },
      },
      include: {
        posts: {
          where: { name: 'createdPostName' }
        }
      }
    });
```

If you'd like this feature by default in Prisma, then you could create a feature request for this use case [here](https://github.com/prisma/prisma/issues/new?assignees=&labels=&template=feature_request.md&title=)

**Reply by ranst91** [reply]
My post object has "content" property, which is not unique, therefore, If I create another post with the exact same content, the query above will return two posts, which leads to not being able to tell which one is the last created one.

Isn't there any method to specify "I want the latest created"?

**Reply by ryands17** [reply]
Not at the moment unfortunately. You would need to query by something unique for now.

**Reply by yunfan** [reply]
> Not at the moment unfortunately. You would need to query by something unique for now.

can people create the Post directly?

**Reply by ryands17** [reply]
> can people create the Post directly?

Yes you can use `prisma.post.create` for that.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
